### PR TITLE
Added aws costs plugin

### PR DESCRIPTION
**Pull request by jdesouza**
## Checklist

* [ ] I have updated the CHANGELOG of any plugins updated
* [ ] I have updated the version file of any plugins updated
* [ ] I have added tests wherever appropriate
* [ ] I have labeled this PR with a label per plugin updated

### CODE REVIEW

File: plugins/aws-costs/build.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PLUGIN_NAME='aws-costs'
+REPOSITORY_NAME=fairwinds/fw-aws-costs
```
Comment (rbren) [inline plugins/aws-costs/build.config:2]: let's just call it aws-costs

Comment (jdesouza) [reply plugins/aws-costs/build.config]: Fixed

File: plugins/aws-costs/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template 
+
+This is a template plugin. When adding a new plugin you can start with copying this directory and changing the values from there.
```
Comment (rbren) [inline plugins/aws-costs/README.md:3]: we should change this readme 😄

File: plugins/aws-costs/aws-costs.sh [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    resource_tags_user_kubernetes_cluster='$cluster' \
+    AND line_item_usage_end_date > timestamp '$initial_date_time' \
+    AND line_item_usage_end_date <= timestamp  '$final_date_time' \
+    GROUP BY  1"
+--work-group "cur_athena_workgroup" \
+--query-execution-context Database=athena_cur_database,Catalog=AwsDataCatalog)
+
+executionId=$(echo $queryResults | jq .QueryExecutionId | sed 's/"//g')
+
+for i in 1 2 3 4 5 6 7 8 9 10; do
```
Comment (rbren) [inline plugins/aws-costs/aws-costs.sh:60]: I think we should make this configurable - we usually have a `timeout` param that gets passed in.

You can use `seq $timeout` to generate the list, e.g. https://www.cyberciti.biz/faq/bash-for-loop/#seq_command

File: plugins/trivy/version.txt [hunk 4]
```diff
@@ -1 +1 @@
-0.13.0
+0.14.0
```
Comment (rbren) [inline plugins/trivy/version.txt:1]: let's make this a patch

File: plugins/aws-costs/results.schema [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "required": [
+    ],
+    "properties": {
+    }
+}
```
Comment (rbren) [inline plugins/aws-costs/results.schema:8]: I'd love to see the actual schema here. What does the output look like?

This is in http://json-schema.org/

File: plugins/aws-costs/aws-costs.sh (hunk 3 above)
Comment (jdesouza) [reply plugins/aws-costs/aws-costs.sh:60]: nice

File: plugins/aws-costs/results.schema (hunk 5 above)
Comment (jdesouza) [reply plugins/aws-costs/results.schema:8]: cool

File: plugins/aws-costs/README.md (hunk 2 above)
Comment (jdesouza) [reply plugins/aws-costs/README.md:3]: done

File: plugins/aws-costs/aws-costs.sh (hunk 3 above)
Comment (jdesouza) [reply plugins/aws-costs/aws-costs.sh:60]: done

File: plugins/aws-costs/aws-costs.sh [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+      line_item_product_code, identity_time_interval, round(sum("line_item_unblended_cost"),2) AS cost \
+    FROM 
+      "$database"."$table"  
+    WHERE \
+      resource_tags_user_$tagkey='$tagvalue'
+      AND line_item_usage_end_date > timestamp '$initial_date_time' \
+      AND line_item_usage_end_date <= timestamp '$final_date_time' \
+    GROUP BY  1,2" \
+--work-group "cur_athena_workgroup" \
+--query-execution-context Database=athena_cur_database,Catalog=AwsDataCatalog)
```
Comment (rbren) [inline plugins/aws-costs/aws-costs.sh:84]: need to use $database here I think. Do we need to parameterize AwsDataCatalog?

**Review by rbren — APPROVED** [review]

**Pull request merged**